src/sybild/ingest.go

```diff
@@ -0,0 +1,39 @@
+package sybild
+
+import (
+	"bytes"
+	"io"
+	"os"
+	"os/exec"
+
+	"github.com/davecgh/go-spew/spew"
+)
+
+func sybilIngest(tableName string, buf io.Reader) error {
```

Comment:
i can't believe how much code exists for simple shell things, but i imagine people must love how it looks because they don't know any better